cli/command/stack/kubernetes/loader.go

```diff
@@ -1,20 +1,23 @@
 package kubernetes
 
 import (
+	"io"
+
 	"github.com/docker/cli/cli/compose/loader"
 	composetypes "github.com/docker/cli/cli/compose/types"
 	yaml "gopkg.in/yaml.v2"
 )
 
-func loadStack(name string, cfg composetypes.Config) (stack, error) {
+func loadStack(stderr io.Writer, name string, cfg *composetypes.Config) (stack, error) {
```

Comment:
nit: any reason for having `stderr` as first argument ? 😝

Reply:
For homogenization 😇 ?
https://github.com/docker/cli/blob/master/cli/command/stack/swarm/deploy_bundlefile.go#L100

Reply:
👍